src/DynamoCore/Logging/DynamoAnalyticsClient.cs

```diff
@@ -151,7 +151,7 @@ public void Start(DynamoModel dynamoModel)
             var appversion = dynamoModel.AppVersion;
 
             //If not ReportingAnalytics, then set the idle time as infinite so idle state is not recorded.
-            Service.StartUp(new ProductInfo() { Name = "Dynamo", VersionString = appversion },
+            Service.StartUp(new ProductInfo() { Name = "Dynamo", VersionString = appversion, AppVersion = dynamoModel.Version },
                 new UserInfo(Session.UserId), ReportingAnalytics ? TimeSpan.FromMinutes(30) : TimeSpan.MaxValue);
```

Comment:
what is the diff b/w appversion and dynamoModel.version?

Reply:
`DynamoModel.Version` is version of Dynamo. e.g.
`DynamoModel.AppVersion` is the application version string for analytics reporting APIs. e.g. "DynamoSandbox.vshost-1.3.3.3046"

Reply:
ok..shouldn't it say coreversion?

Reply:
This change may not be needed in Dynamo at all, currently it's crashing Dynamo since version is not available at this moment. Without passing this result in not using some new functionality in Analytics but should be consistent with RC1.3.2. I will revert it